### DISCUSSION TITLE
can anyone help us to sign the build apps .

### DISCUSSION THREAD

**Discussion by ajithjojo** [NativePHP/laravel#335] [Q&A]
I was trying to make sign a build , but the documentation was not enough , can anyone guide us to sign the build , windows and ios , (mainly windows )

**Comment by simonhamp** [comment]
You won't be able to build for iOS with Electron.

Windows build signing docs are coming. I will update here once they're ready

I've raised an [issue](https://github.com/NativePHP/laravel/issues/361) to track this
